CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.3.8 (Next)
 * [#72](https://github.com/ashkan18/graphlient/pull/72): Add http_options - [@neroleung](https://github.com/neroleung).
 * [#71](https://github.com/ashkan18/graphlient/issues/70): Add `Graphlient::Errors::TimeoutError` - [@BenDrozdoff](https://github.com/BenDrozdoff).
+* [#75](https://github.com/ashkan18/graphlient/pull/75): Fix faraday use / adapter error - [@jfhinchcliffe](https://github.com/jfhinchcliffe).
```

Comment:
This is really "Add support for Faraday X.Y", forget which version did this... isn't it?